src/QuadForm/Herm/Genus.jl

```diff
@@ -1528,15 +1563,21 @@ Given a global genus symbol `G` for hermitian lattices over $E/K$, return a herm
 lattice over $E/K$ which admits `G` as global genus symbol.
 """
 function representative(G::GenusHerm)
+  if !is_integral(G)
+    s = denominator(scale(G))
+    L = representative(rescale(G, s))
```

Comment:
The way I have defined scales for `GenusHerm`, it is always an ideal in the fixed field so here it should be ok. I called it scale to be close to what we have for `ZGenus` but it is not really the scale of a lattice in this genus (maybe the norm, or a "nice" multiple of it)

Reply:
If it is not a scale, then it should not be called scale.

Reply:
Mhh looking at the code looks like it is the scale?

Reply:
Normally the scale is an ideal in the base field, here since I take products of power of ideals in the fixed field, a priori it is not... except if the scale is fixed by the involution, in which case what I compute is the intersection with the fixed ring